Neighborly/VectorList.cs

```diff
@@ -147,42 +125,7 @@ public List<Vector> GetAllVectors()
         return _vectorList.ToList();
```

Comment:
ToList() returns the Vectors that exist in vectorList. 
This need not be fixed, however. I want to remove the ToList() function and advocate for using the enumerator to create a new copy of Vectors if clients really need that.

Reply:
Yup, I noticed that. Decided not to change this, as it didn't return the in-memory-items before, either. This would be problematic if #33 gets implemented, anyways.